lib/engine/game/g_18_eus/step/bidbox_auction.rb

```diff
@@ -5,128 +5,130 @@
 module Engine
   module Game
     module G18EUS
-      module BidboxAuction
-        include Engine::Step::Auctioner
+      module Step
+        module BidboxAuction
+          include Engine::Step::Auctioner
+
+          def actions(entity)
+            actions = []
+            actions << 'bid' << 'pass'
+            actions.concat(super).uniq
+          end
+
+          def setup
+            # This sets the initial value of @bids
+            setup_auction
+            super
+
+            @bid_actions = 0
+            @bids = @round.bids if @round.bids
+            # Set initial value of @bids on the round if there's none.
+            @round.bids = @bids unless @round.bids
+          end
+
+          def can_buy_company?(_player, _company)
+            false # Only companies are privates
```

Comment:
Should bidbox know this or should it be in the step itself?

Reply:
Probably in the step.  Moved.